### ISSUE TITLE
Stories not saved on fresh "install"

### ISSUE THREAD

**Issue by stohrendorf** [klembot/twinejs#511]
Twine 2.2.1, Linux x64, downloaded from homepage

I'm currently "not amused". After writing for about an hour, I closed Twine, then reopened it, and my story is gone. There is no twine folder anywhere except for the application itself. There were no error messages or message boxes warning me that the data could not be saved. Twine itself detected the locale as German (which is correct), but didn't save anything in `$HOME/Dokumente` (which is correctly reported by `xdg-user-dir DOCUMENTS`), so I assume somewhere `$HOME/Documents` or the like is hardcoded. I'm questioning how it is even possible to not detect this sort of error leading to data loss.

**Comment by klembot** [comment]
First of all, I'm sorry you had a bad experience here. The name of the Documents folder should be localized, but it is possible that something went wrong.

Could you try the following steps to help me narrow down what went wrong:
- What distribution of Linux are you using?
- Please re-run Twine again and press control-shift-alt-D. This should bring up a debugger window. Are any errors reported there when you first start up?
- Could you do a filesystem search, if you haven't already, just to make sure it did not save your files somewhere unexpected? Doing a search for "Twine" should work, or possibly the name of the story you created.
- Finally, if you would not mind trying the [latest beta](https://github.com/klembot/twinejs/releases/tag/2.3.0-beta1) to see if it behaves better, that would be much appreciated. Version 2.3.0 uses a different desktop app wrapper library, which might handle your particular situation better.

**Comment by stohrendorf** [comment]
Distro is opensuse leap 15. Debug console and stdout was empty. Filesystem search found nothing outside of .cache and .config and the application itself.

The beta shows some DOMExceptions, when initially creating a story, but nothing about the filesystem; stdout says `Failed to fully publish story. Attempting naked publish.` when creating a new story. It still does not save the stories.

**Comment by stohrendorf** [comment]
This seems to be a duplicate of #386 and #335, partially #354, and https://github.com/klembot/twinejs/issues/340#issuecomment-402375841 seems to be the root cause unless that's already changed.

**Comment by klembot** [comment]
Hi, could you let me know if beta 2 resolves this? I would also try using the "Show Story Library" menu item in the window to see where Twine thinks it is saving stories.

**Comment by anon3000** [comment]
I found this issue after a little searching because I was experiencing the same issue with Twine 2.2.21 Linux x64, where Twine was not saving stories, nor creating the Twine directory in the documents folder.

I don't know if this is noteworthy, but my distro is also OpenSuse Leap 15. My locale is English, but my documents folder is "~/documents" rather than "~/Documents", although `xdg-user-dir DOCUMENTS` reports the correct path.

After downloading the Twine 2.3.0 beta2 release, I found that this issue was no longer present. Twine created the appropriate Twine and Stories directories. Stories are saved as normal. "Show Story Library" pulls up the stories directory, while on 2.2.21 it would do nothing.

If OP's issue is the same as my own, which I believe it was, then Beta 2's changes have resolved it.
